### DISCUSSION TITLE
Asking help on tool usage

### DISCUSSION THREAD

**Discussion by amcny** [mxrch/GHunt#410] [General]
Hello, please help me to use this tool,

The problem is I didn't understand how to use this tool with given steps, I didn't understand ghunt login, could please explain step by step with patience please, thanks in advance, love from India

**Comment by sk3lk0** [comment]
i got problem with login, tool asked me for enter oauth login but i didn't find him in cookie's list, i decided to use ghunt extension, it helped me

**Comment by amcny** [comment]
> i got problem with login, tool asked me for enter oauth login but i didn't find him in cookie's list, i decided to use ghunt extension, it helped me

Hey Hello, thanks for your response, I want complete guide how to use this tool, I want to contact you, shall I..??

**Comment by amcny** [comment]
Please Guide me , how to use this tool step by step process

**Comment by mxrch** [comment]
Everything is in the Read Me. They are commands to type in the terminal.
Idk how to explain more.

If you don't understand it, you should first learn how to use a terminal.

**Reply by amcny** [reply]
First, launch the listener by doing ghunt login and choose between 1 of the 2 first methods :

👆 I didn't understand this, sorry if you get angry, I'm noob and beginner, please have patience and guide me, thanks in advance

**Reply by mxrch** [reply]
Do `ghunt login` in your terminal

**Reply by amcny** [reply]
when I typed this ghunt login, its shows nothing

**Reply by mxrch** [reply]
Did you installed GHunt first ?
https://github.com/mxrch/GHunt#%EF%B8%8F-installation

**Reply by amcny** [reply]
I cloned it using git clone <url> command

**Reply by mxrch** [reply]
You don't have to, just do the commands in the installation section.

**Reply by amcny** [reply]
Thanks a lot, its working

**Comment by amcny** [comment]
Can we access Google Photos through GHunt...?

**Reply by micduffy** [reply]
Here is a link to the info you need https://letmegooglethat.com/?q=ghunt+photos
